src/webots/nodes/WbFog.cpp

```diff
@@ -142,13 +142,15 @@ void WbFog::updateVisibilityRange() {
 void WbFog::applyChangesToWren() {
   assert(isFirstInstance());
 
+  WrSceneFogType fogType = mWrenFogType;
   float density = 0.0f;
   if (mVisibilityRange->value() > 0.0)
     density = 1.0 / mVisibilityRange->value();
-
+  else
+    fogType = WR_SCENE_FOG_TYPE_NONE;
```

Comment:
This makes perfectly sense but is not 100% coherent with the doc, as in the doc we say it should be 0.0 and not <= 0.0 (we should probably improve the doc).

Reply:
Negative values are not supported and if the visibility value is negative it is automatically set to 0. So it should not be possible that `mVisibilityRange` is less than 0.0.
https://github.com/omichel/webots/blob/75eca6004007b2d3b865e6a0aafffc9d6a2aba18/src/webots/nodes/WbFog.cpp#L135

In the doc it is already specified thant visibilityRange is in the range from 0 to inf, so it doesn't seem necessary to change it.